<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Paaikkuna</class>
 <widget class="QMainWindow" name="Paaikkuna">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>667</width>
    <height>386</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Yhtälönratkaisija</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QLabel" name="otsikko">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>131</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>2. asteen yhtälön ratkaisija</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_a">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>70</y>
      <width>47</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;a&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_b">
    <property name="geometry">
     <rect>
      <x>250</x>
      <y>70</y>
      <width>20</width>
      <height>20</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;b&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_c">
    <property name="geometry">
     <rect>
      <x>490</x>
      <y>70</y>
      <width>31</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:12pt;&quot;&gt;c&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="value_a">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="value_b">
    <property name="geometry">
     <rect>
      <x>290</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="value_c">
    <property name="geometry">
     <rect>
      <x>530</x>
      <y>70</y>
      <width>113</width>
      <height>20</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text_ratkaisu">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>340</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Yhtälön ratkaisu(t):</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_diskr">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>250</y>
      <width>171</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;Diskriminantti:&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_x1">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>330</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;x=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="value_x1">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>330</y>
      <width>131</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>357</x>
      <y>120</y>
      <width>171</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>0</number>
    </property>
   </widget>
   <widget class="QLabel" name="text_or">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>330</y>
      <width>21</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;V&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLabel" name="text_x2">
    <property name="geometry">
     <rect>
      <x>360</x>
      <y>330</y>
      <width>41</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;x=&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLCDNumber" name="value_x2">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="text_d">
    <property name="geometry">
     <rect>
      <x>160</x>
      <y>250</y>
      <width>61</width>
      <height>31</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:20pt;&quot;&gt;D = &lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="solvebutton">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>110</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Ratkaise</string>
    </property>
   </widget>
   <widget class="QLabel" name="diskriminantti">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>250</y>
      <width>151</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>TextLabel</string>
    </property>
   </widget>
   <widget class="QLabel" name="errorlabel">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>160</y>
      <width>311</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>12</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-size:16pt;&quot;&gt;TextLabel&lt;/span&gt;&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>330</y>
      <width>75</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Lopeta</string>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections>
  <connection>
   <sender>pushButton</sender>
   <signal>clicked()</signal>
   <receiver>Paaikkuna</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>631</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>618</x>
     <y>275</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
